app/mongobee/changesets/DeclarationsChangelog.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mongobee.changesets;
+
+import com.github.mongobee.changeset.ChangeLog;
+import com.github.mongobee.changeset.ChangeSet;
+import com.mongodb.DB;
+
+@ChangeLog
+public class DeclarationsChangelog {
+    private String collection = "declarations";
+
+    @ChangeSet(order = "001", id = "External Movements DB Baseline", author = "Paulo Monteiro")
+    public void dbBaseline(DB db) {
+    }
+}
```

Comment:
Is this name correct? I mean `DeclarationsChangelog` and string `collection = "declarations"`

Reply:
:man_facepalming:

Reply:
Resolved. Thank you! :)